godot-codegen/src/util.rs

```diff
@@ -58,12 +58,20 @@ pub fn make_enum_definition(enum_: &Enum) -> TokenStream {
         None
     };
 
+    let mut derives = vec!["Copy", "Clone", "Eq", "PartialEq", "Debug", "Hash"];
+
+    if enum_.is_bitfield {
+        derives.push("Default");
+    }
+
+    let derives: Vec<_> = derives.into_iter().map(ident).collect();
```

Comment:
`collect()` should not be necessary, I think the expression inside `quote!` can also work on an iterator.